<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_Manage_Alts</class>
 <widget class="QWidget" name="dlg_Manage_Alts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Alts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>Picture</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fail Stack</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cmdAdd">
        <property name="text">
         <string>Add Alt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdImport">
        <property name="text">
         <string>Import From Game Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdRemove">
        <property name="text">
         <string>Remove Alt</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>314</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
